risk-map/yaml/controls.yaml

```
@@ -172,6 +209,25 @@ controls:
       - MST
       - MXF
       - MDT
+    mappings:
```

What about `Restrict Library Loading` as a mitre control? 


```suggestion
    mappings:
      mitre-atlas:
        - AML.M0011
```